### ISSUE TITLE
DetectNet & Tensorflow

### ISSUE THREAD

**Issue by tamasbalassa** [NVIDIA/DIGITS#1881]
Hello,

I am very new at Tensorflow (I was using caffe detectnet models in python successfully to detect objects) and I was wondering if there is the DetectNet architecture ( [https://raw.githubusercontent.com/NVIDIA/caffe/caffe-0.15/examples/kitti/detectnet_network.prototxt](url) ) available for creating a Tensorflow model in DIGITS.

If so, then is there any kind of a tutorial or an example / dummy code about how should I load in the pretrained network and visualize the bounding boxes?

**Comment by 00krishna** [comment]
@tamasbalassa did you ever hear anything further about this, or find any good examples? I am looking at the same thing, but just started. Thanks.

**Comment by tamasbalassa** [comment]
@00krishna  no, not really. I guess it shouldnt be that hard to convert it, it would only require some time.
